layouts/partials/hooks/body-end.html

```diff
@@ -1,4 +1,5 @@
 {{ if hugo.IsProduction -}}
-  {{ partial "script.html" (dict "src" "js/tracing.js") -}}
+{{ partial "script.html" (dict "src" "js/tracing.js") -}}
 {{ end -}}
 {{ partial "script.html" (dict "src" "js/navScroll.js") -}}
+{{ partial "script.html" (dict "src" "js/scrollToTop.js") }}
```

Comment:
this throws some errors right now since the `scrollToTopBtn` html element is only on the search-form.html page, we need to find a place where we put it across the page.

Can you also move it into the `body-end.html` for now so we can see it in action, then we can discuss where we want to position it.

cc @chalin

Reply:
The best place to implement it so it’s applied on all pages is the footer.html page which is in the theme/docsy which led to the submodule errors initially in the old pr. @svrnm

Reply:
> The best place to implement it so it’s applied on all pages is the footer.html page which is in the theme/docsy which led to the submodule errors initially in the old pr. @svrnm

We can not edit files in that folder, but we can leverage the body-end hook as outlined in the docsy docs here: https://www.docsy.dev/docs/adding-content/lookandfeel/#customizing-templates

If this works well we can talk about contributing this upstream to docsy, but for now let's focus on our repo

Reply:
Got it, thanks!

Reply:
Yup! you were absolutely right! that was fun

Reply:
@chalin